ui/mainwindow.cpp

```diff
@@ -1025,6 +1025,7 @@ void MainWindow::drawGraph() {
     resetScene();
     g_assemblyGraph->resetNodes();
     g_assemblyGraph->markNodesToDraw(scope, startingNodes);
+    g_settings->nodeColorer->setGlobalScope(scope,g_assemblyGraph->m_firstQuartileDepth, g_assemblyGraph->m_thirdQuartileDepth);
```

Comment:
This does not look right. Colorers are transient, they are recreated as necessary, so, on colorer switch this will be lost.